src/components/vote/Dropdown.jsx

```diff
@@ -1,28 +1,27 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
Can we change this to a js file instead of a jsx file for consistency?

Reply:
sure can!